test/js/reducers/questionnaire.spec.js

```diff
@@ -1176,4 +1188,11 @@ const questionnaire = deepFreeze({
   defaultLanguage: 'en',
   languages: ['en'],
   quotaCompletedMsg: null
-})
+}
+
+// TODO: investigate why Flow ignores the result of `deepFreeze`
+// It probably is defined as `any` somewhere.
+// As a workaround, we define `bareQuestionnaire` and explicitly annotate it as
+// Questionnaire. That will let us catch inconsistencies when we define the
+// Questionnaire fixture for testing here.
```

Comment:
Add the following in a new comment line: "The limitations of `deepFreeze` are probably related to sealed objects being used under its hood. See: https://flowtype.org/docs/objects.html#sealed-object-types"